src/plugin/memory-webview-main.ts

```diff
@@ -38,23 +39,20 @@ import {
     showAdvancedOptionsType,
     StoreMemoryArguments,
     storeMemoryType,
+    ViewState,
+    viewStateChangedType,
     WebviewSelection,
     WriteMemoryArguments,
     WriteMemoryResult,
     writeMemoryType,
 } from '../common/messaging';
 import { getVisibleColumns, WebviewContext } from '../common/webview-context';
-import { AddressPaddingOptions, MemoryViewSettings, ScrollingBehavior } from '../webview/utils/view-types';
+import type { MemoryViewSettings, ScrollingBehavior } from '../webview/utils/view-types';
```

Comment:
The annotation here is correct, but the fact that it's correct suggests that these types should be moved to a common area so that we don't have to guard against accidentally importing webview code inappropriately.

Reply:
I 100% agree that it is a good idea to actually move the necessary interfaces into a common area, will do, thanks!